src/System.Xml.XDocument/src/System.Xml.XDocument.csproj

```diff
@@ -21,6 +21,9 @@
     <Compile Include="$(CommonPath)\System\SR.cs">
       <Link>Resources\SR.cs</Link>
     </Compile>
+    <Compile Include="$(CommonPath)\System\IO\StringBuilderCache.cs">
+      <Link>System\StringBuilderCache.cs</Link>
```

Comment:
Agreed.  I tried to remain consistent in the .csproj with however that .csproj was doing it, but it'd be good to be consistent across the repo.